src/renderer/components/LandingPage/HeaderView.vue

```diff
@@ -133,12 +133,12 @@ export default {
       }
     }
     .faExternal {
-      color: $white-color;
+      color: $pitch;
       font-size: $default-font-size;
     }
 
     .faRefresh {
-      color: $white-color;
+      color: $pitch;
```

Comment:
Have you double checked this? Changing from `white` to `dark` is a bit risky.

Reply:
good catch!